packages/suite/src/utils/suite/__tests__/guide.test.ts

```diff
@@ -1,31 +1,38 @@
 import * as guideUtils from '../guide';
 import * as fixtures from '../__fixtures__/guide';
 
-describe('getNodeTitle', () => {
+/*
+ * A test file that contains tests that are dependent on current structure of guide.
+ * As the structure and articles of the guide are constantly changing, this test would have to be constantly updated.
+ * For now, the tests are skipped to avoid wasting time.
+ * GitHub issue #4585 has been created. It's goal is to create generic tests.
+ */
+
+describe.skip('getNodeTitle', () => {
```

Comment:
this was not the culprit here. this one is `packages/integration-tests/projects/suite-web/tests/suite/guide.test.ts`